package/MDAnalysis/analysis/gnm.py

```diff
@@ -276,6 +277,12 @@ def _generate_output(self, w, v, outputobject, time, matrix,
         # outputobject.append((time, [ w[list_map[i]] for i in range(nmodes) ],
         # [ v[list_map[i]] for i in range( nmodes) ] ))
 
+        warnings.warn(
```

Comment:
I think it would be best to have some kind of docstring entries to let users know what's going on. Thoughts?

Reply:
I put an italic note to the attribute descriptor. What do you think?

Reply:
Awesome thanks :) !